packages/create-plugin/templates/github/ci/.github/workflows/ci.yml

```diff
@@ -74,16 +74,30 @@ jobs:
       - name: Check for E2E
```

Comment:
Not related to this PR but since you are already extensively testing it. We should change
```
- uses: pnpm/action-setup@v2
```
to
```
- uses: pnpm/action-setup@v3
```
to make it use node 20

Reply:
Thanks. Tis done!